integ-tests/src/main/java/com/google/pojo/UserTableRow.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.pojo;
+
+public class UserTableRow {
```

Comment:
Please document public classes and methods.

Reply:
Will do.

Reply:
Should I document inner classes in the POJO?

Reply:
If they are public, yes please. Self-explanatory members (like getters) don't need javadoc, though. See https://engdoc.corp.google.com/eng/doc/devguide/java/style/index.md?cl=head#s7-javadoc

Reply:
Fixed.